packages/wallet-service/src/ws/utils.ts

```diff
@@ -24,15 +25,15 @@ export const connectionInfoFromEvent = (
 ): WsConnectionInfo => {
   const logger: Logger = createDefaultLogger();
   const connID = event.requestContext.connectionId;
-  if (process.env.IS_OFFLINE === 'true') {
+  if (config.isOffline) {
```

Comment:
Wow, I don't remember what this is for

Can't we use `config.stage === 'local'` here as well?

Reply:
Sure, but there seems to be 2 stages for running locally.
- dev: https://github.com/HathorNetwork/hathor-wallet-service/blob/master/packages/wallet-service/src/api/wallet.ts#L95
- local: https://github.com/HathorNetwork/hathor-wallet-service/blob/master/packages/wallet-service/src/utils.ts#L132

Should we change these as well?

Reply:
Yea, I will investigate and normalize this, nothing to do in this current PR